public/views/partials/atoms/badge.liquid

```diff
@@ -0,0 +1,39 @@
+---
+metadata:
+  name: Badge
+  params:
+    tag: "div"
+    maxValue: 1000
+    value: 1000
+  styleguide:
+    - tag: "div"
+      maxValue: 20
+      value: 8
+    - tag: "div"
+      maxValue: 999
+      value: 1500
+    - tag: "div"
+      maxValue: 1999
+      value: 1500
+    - tag: "div"
+      value: 1500
+
+---
+{% liquid
+  assign tag = params.tag | default: 'div'
+  assign maxValue = params.maxValue | default: 0
+  assign value = params.value
+
+  if maxValue > 0 and maxValue < value
+    assign value = params.value | at_most: maxValue | append: "+"
```

Comment:
if `maxValue < value` then `value` should always be `maxValue | append: "+"` -- so i think there is no need for `params.value | at_most`